ext/reflection/php_reflection.stub.php

```diff
@@ -18,6 +18,9 @@ interface Reflector extends Stringable
 
 abstract class ReflectionFunctionAbstract implements Reflector
 {
+    /** @var string */
+    public $name = "";
```

Comment:
This property used to be registered as abstract. Since it doesn't seem to me that we validate the implementation of abstract properties anywhere, I got rid of the `ZEND_ACC_ABSTRACT` flag.